src/infra/internalDatabase/model/account.go

```diff
@@ -45,8 +46,17 @@ func (model Account) ToEntity() (accountEntity entity.Account, err error) {
 		return accountEntity, err
 	}
 
+	secureAccessKeys := []entity.SecureAccessKey{}
+	for _, secureAccessKeyModel := range model.SecureAccessKeys {
+		secureAccessKeyEntity, err := secureAccessKeyModel.ToEntity()
```

Comment:
Here you added the Entity suffix but you didn't add to ANY of the account ones I mentioned (and there are more). Why brother? I'm telling ya, you're two different persons on the same PR. 
![](https://media2.giphy.com/media/v1.Y2lkPTc5MGI3NjExMWtjZHNsNGxkdzNsajE2aWcxb2wwdzI2MmZ2YnpudnZqdm02OWx5ZiZlcD12MV9pbnRlcm5hbF9naWZfYnlfaWQmY3Q9Zw/gGAE8FpSkr8wDYm0Jr/giphy.webp)